src/app/zap-templates/zcl/data-model/chip/color-control-cluster.xml

```diff
@@ -82,6 +82,28 @@ limitations under the License.
     <item name="ActivateFromEnhancedCurrentHue" value="0x02"/>
   </enum>
 
+  <bitmap name="ColorControlOptions" type="bitmap8">
```

Comment:
This does not exist in the spec with this name.

Reply:
use OptionsBitmap

Reply:
updated with OptionsBitmap